src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -476,7 +483,11 @@ export function fetchRequestDetails(id) {
   };
 }
 
-export function fetchConfirmedAppointmentDetails(id, type) {
+export function fetchConfirmedAppointmentDetails(
+  id,
+  type,
+  fetchClaimStatus = false,
```

Comment:
Since this is a single appointment, I'm also happy to have this fetch claims details by default.